cmd/client/wallet/main.go

```diff
@@ -134,10 +142,29 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Enable log if the last parameter is -verbose
-	if os.Args[len(os.Args)-1] == "--verbose" {
-		setupLog()
-		os.Args = os.Args[:len(os.Args)-1]
+ARG:
+	for {
+		lastArg := os.Args[len(os.Args)-1]
+		switch lastArg {
+		case "--verbose":
+			setupLog()
+			os.Args = os.Args[:len(os.Args)-1]
+		case "--devnet":
+			// the multiaddress of bootnodes for devnet
+			addrStrings = []string{"/ip4/100.26.90.187/tcp/9871/p2p/Qmdfjtk6hPoyrH1zVD9PEH4zfWLo38dP2mDvvKXfh3tnEv", "/ip4/54.213.43.194/tcp/9871/p2p/QmRVbTpEYup8dSaURZfF6ByrMTSKa4UyUzJhSjahFzRqNj"}
+			os.Args = os.Args[:len(os.Args)-1]
+		default:
+			break ARG
+		}
+	}
+
+	if len(os.Getenv("RpcNodes")) > 0 {
+		rpcServers = utils.StringsToPeers(os.Getenv("RpcNodes"))
+	}
+	if len(rpcServers) == 0 {
+		fmt.Println("Error: please set environment variable RpcNodes")
```

Comment:
nit: have a const for env var string, so that we can refer to it and keep consistant.